website/docs/documentation/Pipelines/Optimus_Pipeline/README.md

```diff
@@ -235,3 +254,17 @@ The Optimus pipeline is a single sample pipeline, but it can accept multiple FAS
 
 Parameters are listed in each task WDL. For a list of the tasks, see the table in the [Task Summary Section](#optimus-task-summary). Select the link for the task of interest and then view the parameters in the task WDL "command {}" section. For the task Docker image, see task WDL "# runtime values" section; the Docker is listed as "String docker =  ". If you want to learn more about all the different parameters available for a software tool, please select the relevant link in the table's "Tool" column.
 :::
+
+::: details How does Optimus compare to Cell Ranger? 
+
+Cell Ranger is a commonly used set of analysis pipelines developed by [10x Genomics](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger). Optimus and Cell Ranger share many features and additionally, Optimus results are validated against Cell Ranger results (see our [human validation report](https://github.com/broadinstitute/warp/blob/master/pipelines/skylab/optimus/benchmarking/v1_Apr2019/optimus_report.rst)). 
+
+*So why develop an independent pipeline for 10x data analyses?* 
+
+For three reasons:
+1) Need for an open source, cloud-optimized pipeline. When Optimus was developed, Cell Ranger software was not yet open source, nor was it optimized for the cloud. We did not want to limit software usage to what was available by 10x, but rather wanted  to harness the breadth of tools available in the scientific community. 
```

Comment:
I think I want to make it clearer here that CellRanger is still not open source. They have not updated the content of their open source code base since August 2017, yet have published multiple versions of CellRanger in the meantime.

Reply:
Updated this description.